docs/dqx/src/pages/index.tsx

```diff
@@ -145,6 +145,14 @@ const Hero = () => {
           label="User guide"
           className="w-full md:w-auto"
         />
+        <Button
+          variant="secondary"
+          outline={true}
+          link="/docs/demos"
+          size="large"
+          label="Demos"
+          className="w-full md:w-auto"
+        />
```

Comment:
5 buttons on the intro page is just too much. I'll later add them to navbar, fine for now.